packages/example-getting-started/docs/putting-it-together.md

```diff
@@ -26,26 +26,31 @@ other artifacts and inject them into our application for use.
 #### src/application.ts
 ```ts
 import {ApplicationConfig} from '@loopback/core';
-import {RestApplication} from '@loopback/rest';
+import {RestApplication, RestServer} from '@loopback/rest';
+import {MySequence} from './sequence';
 import {db} from './datasources/db.datasource';
 
 /* tslint:disable:no-unused-variable */
-// Do not remove!
-// Class and Repository imports required to infer types in consuming code!
 // Binding and Booter imports are required to infer types for BootMixin!
 import {BootMixin, Booter, Binding} from '@loopback/boot';
 import {
   Class,
   Repository,
   RepositoryMixin,
+  juggler,
```

Comment:
We also need a notice for these two imports, similar to the notice given for `Binding` and `Booter`